ui/hooks/src/use-reasons.ts

```diff
@@ -6,7 +6,7 @@ import { getModerationReasons, Reason } from './moderation-requests';
 
 export interface UseReasonsActions {
   /**
-   *  fetch reasons, pass active as true to get only active reasons
+   * fetch reasons, pass active as true to get only active reasons
    */
   fetchReasons: (data: { active: boolean }) => void;
```

Comment:
you said to `get`. But the return of this method is `void`. It's correct?

Reply:
yes, the action method just dispatches a payload for the reducer. the hook itself however returns the state and the action.
The active flag was because some reasons maybe flagged as inactive in the db. The flag ensures that only currently active reasons were being fetched and not everything else.